src/app/pages/wallets/wallet-details/pages-wallet-details.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container walletDetailsContainer">
+
+  <div class="wallet__description">
+    <div class="name">
+      <div class="name__name">
+        {{ walletsDetailsData.data?.name }}
+      </div>
+      <div class="name__edit">
+        <button mat-stroked-button color="primary" (click)="handleWalletEdit(walletsDetailsData.data!)">Edit</button>
+      </div>
+    </div>
+
+    <div class="newTransaction">
+      <button mat-flat-button color="primary">Add new transaction</button>
+    </div>
+  </div>
+
+  <div class="wallet__sort">
+    <mat-form-field appearance="fill">
+      <mat-label>All Transactions</mat-label>
+      <mat-select [(value)]="selectedTransactionType" (selectionChange)="onTransactionTypeChange($event)">
```

Comment:
If you will go with `[formControl]` approach instead two-way binding `[( )]` you can just listen to changes on the `FormControl` via `.valueChanges` and filter out displayed transactions. That way this `transactionsTypeFilter` pipe will be obsolete.